README.md

```diff
@@ -132,10 +132,109 @@ is not supported anymore. Exporting `$ROLENAME_provider_os_default` allows the u
 in Ansible) and thus get a consistent setting for all the systems of the given OS version without
 having to decide what the actual value is - the decision is delegated to the role.)
 
-
 Implementation considerations
 ===========
 
+## Role Structure
+Avoid testing for distribution and version in tasks. Rather add a variable file to "vars/"
+for each supported distribution and version with the variables that need to change according
+to the distribution and version. This way it is easy to add support to a new distribution by
+simply dropping a new file in to "vars/", see below
+[Supporting multiple distributions and versions](#supporting-multiple-distributions-and-versions). See also
+[Vars vs Defaults](#vars-vs-defaults) which mandates "Avoid embedding large lists or "magic values" directly
```

Comment:
use single quotes around `magic values` since it is inside a double quoted string - `'magic values'`

Reply:
ha! true.